css/style.css

```diff
@@ -246,10 +248,49 @@ a img{
     background-image: url("../img/porty.JPG");
 }
 
-/*#contact{*/
-    /*background-color: #0C2C40;*/
-    /*height:200px;*/
-/*}*/
+#contact{
+    height:350px;
+}
+
+.contact-container{
+    width:80%;
+    margin:100px auto;
+}
+
+.contact-area{
+    width:16.6%;
+    float:left;
+}
+
+.contact-pic{
+    width:50px;
+    height:50px;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    margin: 0 auto;
+    display:block;
+}
+
+.contact2{
```

Comment:
I don't this these classes are referenced anywhere in your code... you maverick!